src/platform-apps/channels/channel-view.tsx

```diff
@@ -79,13 +93,19 @@ export class ChannelView extends React.Component<Properties> {
   render() {
     return (
       <div className='channel-view'>
+        <IndicatorMessage
+          hasNewMessage={this.props.hasNewMessage}
+          closeIndicator={this.props.closeIndicator}
+          scrollToBottom={this.scrollToBottom}
```

Comment:
i think we can get rid of one of these callbacks. let's keep the 'closeIndicator' one and remove the scrollToBottom one, as the Indicator component shouldn't need to know what the ultimate effect of closing it will be.

can we also add a test for this? it should make sure that we're scrolling the ref, as well as forwarding the callback to the parent.